app/src/main/java/org/mozilla/FirefoxLite/firefoxdownloader/MainActivity.java

```diff
@@ -0,0 +1,294 @@
+
+
+import java.util.Calendar;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.app.ActivityCompat;
+import androidx.core.app.NotificationCompat;
+import androidx.core.content.ContextCompat;
+import androidx.fragment.app.DialogFragment;
+import androidx.lifecycle.LifecycleOwner;
+
+public class MainActivity extends AppCompatActivity implements TimePickerDialog.OnTimeSetListener {
+
+
+    final String URL1 = "http://mattmahoney.net/dc/enwik8.pmd";
+    final String URL2 = "http://mattmahoney.net/dc/enwik9.pmd";
+    final String URL3 = "http://www.appsapk.com/downloading/latest/WeChat-6.5.7.apk";
+    final String URL4 = "http://www.appsapk.com/downloading/latest/Emoji%20Flashlight%20-%20Brightest%20Flashlight%202018-2.0.1.apk";
+    final String URL5 = "http://www.appsapk.com/downloading/latest/Screen%20Recorder-7.7.apk";
+    final String URL6 = "http://www.appsapk.com/downloading/latest/Call%20Recorder%20-%20Automatic%20Call%20Recorder-1.6.0.apk";
+    final String URL7 = "http://www.appsapk.com/downloading/latest/Sound%20Profile%20(+%20volume%20scheduler)-5.25.apk";
+    final String URL8 = "http://www.appsapk.com/downloading/latest/Evernote%20-%20stay%20organized.-7.9.7.apk";
+    final String URL9 = "http://www.appsapk.com/downloading/latest/UC-Browser.apk";
+    final String URL10 = "http://www.appsapk.com/downloading/latest/Barcode%20Scanner-1.2.apk";
+    final String URL11 = "http://download.blender.org/peach/bigbuckbunny_movies/BigBuckBunny_640x360.m4v";
+    final String URL12 = "http://www2.sdfi.edu.cn/netclass/jiaoan/englit/download/Harry%20Potter%20and%20the%20Sorcerer's%20Stone.pdf";
+    final String URL13 = "https://media.giphy.com/media/Bk0CW5frw4qfS/giphy.gif";
+    final String URL14 = "http://techslides.com/demos/sample-videos/small.mp4";
+    final String URL15 = "http://www.sample-videos.com/video/mp4/720/big_buck_bunny_720p_10mb.mp4";
+
+    Button buttonOne;
+    Button buttonCancelOne;
+    Button buttonScheduleOne;
+
+
+
+
+    TextView textViewProgressOne,textViewProgressTwo,textViewProgressThree, textViewProgressFour, textViewProgressFive, textViewProgressSix,
+            textViewProgressSeven, textViewProgressEight, textViewProgressNine,
+            textViewProgressTen, textViewProgressEleven, textViewProgressTwelve,
+            textViewProgressThirteen, textViewProgressFourteen, textViewProgressFifteen;
+
+    ProgressBar progressBarOne, progressBarTwo, progressBarThree,
+            progressBarFour, progressBarFive, progressBarSix,
+            progressBarSeven, progressBarEight, progressBarNine,
+            progressBarTen, progressBarEleven, progressBarTwelve,
+            progressBarThirteen, progressBarFourteen, progressBarFifteen;
+
+    MainActivity mainActivity;
+    private int STORAGE_PERMISSIONS_CODE = 1;
+    private static String dirPath;
+    private  NotificationHelper mNotificationHelper;
```

Comment:
Please remove `m` prefix, and give it a more specific name.

Reply:
Removed notification helper as it was not needed when converted to service